main.py

```diff
@@ -2,6 +2,25 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
```

Comment:
The genre 'Dramma' seems to be a typo and is corrected later in the code. Consider using the correct spelling 'Drama' initially to avoid unnecessary updates.